packages/amplication-data-service-generator/src/admin/create-admin.ts

```diff
@@ -26,6 +27,19 @@ import { createDotEnvModule } from "./create-dotenv";
 const STATIC_MODULES_PATH = path.join(__dirname, "static");
 const API_PATHNAME = "/api";
 
+const validatePath = (adminPath: string) => adminPath.trim() || null;
+const dynamicPathCreator = (adminPath: string) => {
```

Comment:
see my other comment about types - and maybe even one type to hold all the parameters both for the server and admin